src/blk/pmem/PMEMDevice.h

```diff
@@ -40,7 +41,7 @@ class PMEMDevice : public BlockDevice {
 public:
   PMEMDevice(CephContext *cct, aio_callback_t cb, void *cbpriv);
 
-
+  bool supported_bdev_label() override { return !devdax_device; };
```

Comment:
Where is this function called?

Reply:
called in Bluefs.cc or BlueStore.cc. For using /dev/dax, it is not a block device. So we cannot allow normal block operations, which is same to use SPDK to drive nvme devices.

Reply:
might want to remove the trailing `;`.

Reply:
ack